components/dashboard/src/repositories/repositories.routes.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export const repositoriesRoutes = {
+    Main: () => "/repositories",
+    Detail: (id: string) => `/repositories/${id}`,
+    Configuration: (id: string) => `/repositories/${id}/configuration`,
+    Prebuilds: (id: string) => `/repositories/${id}/prebuilds`,
+    PrebuildDetail: (id: string, prebuildId: string) => `/repositories/${id}/prebuilds/${prebuildId}`,
```

Comment:
Or, we could also have it nested like you specify here and redirect `/repositories/:id/prebuilds` to `/prebuilds?configurationId=:id`, but that would only  make sense if `/configuration/:id/prebubilds` would not be prebuild settings, but rather a prebuilds list.

Reply:
> (repoId and prebuildId is not related and will change url into /prebuilds/<prebuild_id> when we want to implement single prebuild page)

I wrote this in PR description. We don't need to review dashboard changes, because it's about to test api whether is working or not. Will be removed before merge

Reply:
@filiptronicek  I will resolve all dashboard reviews, let's do it in followup PR 🙏

Reply:
(but keep your reviews in mind and fix it in followup PR for sure XD

Reply:
Cool stuff, thanks!